scripts/gh-safe.sh

```diff
@@ -54,56 +53,13 @@ is_merge_command() {
     return 1
 }
 
-# Function to check if command is trying to fetch PR comments
-is_pr_comment_fetch() {
```

Comment:
take a look at the setup.sh script, we have some regarding pr commetns to remove there too